### DISCUSSION TITLE
Zero plugin how to add the data back when auto tagging ?

### DISCUSSION THREAD

**Discussion by neilanthonystuckey** [beetbox/beets#5015] [General]
I get a lot of split album in Plex.  Trying to solve this issue without having to do a lot of manual intervention.  I did some tests with mp3tag in windows to replace track and disc numbers  on albums that show as split then refresh Plex's metadata and 9/10 it solves the issues.  I am using the chroma, zero and force option when auto tagging.  I set the track and disc ftags to be zero'd and it does. How can i have beets re-apply the track and disc number tags rather than be blank after the zero plugin runs.

**Comment by DjSlash** [comment]
The whole point of the zero plugin is that the metadata is being nullified. First start by using one tool to process files' metadata. When using beets, make sure to select the proper release from musicbrainz. Don't nullify track/disc-numbers and a long as the release isn't split, it'll be fine. But keep in mind that there are a lot of albums that are split over several mediums, so it's not completely avoidable.

**Reply by neilanthonystuckey** [reply]
Thanks for your time. Is there a better approach with beets to somehow force the update/change of the disc and track tags ? It seems beets does not modify them if they are there. Plex appears to work best with just 1,2,3 etc instead of 1//2 or 2//4 etc.

**Reply by DjSlash** [reply]
One thing that can help, is setting [from_scratch](https://beets.readthedocs.io/en/stable/reference/config.html?highlight=from_scratch#from-scratch) in your configuration.
